### PR TITLE
Add tests for untag command and untag command parser

### PR DESCRIPTION
**Pull request by iuhiah**
Fixed bug in untag error message exposed by test (line 66 of `UntagCommand.java`)

### CODE REVIEW

File: src/test/java/seedu/innsync/logic/commands/UntagCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+import seedu.innsync.model.ModelManager;
+import seedu.innsync.model.UserPrefs;
+import seedu.innsync.model.person.Person;
+import seedu.innsync.testutil.PersonBuilder;
+
+public class UntagCommandTest {
+
+    private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+
+    @Test
```
Comment (varsha13152) [inline src/test/java/seedu/innsync/logic/commands/UntagCommandTest.java:27]: The test cases are very comprehensive, I think you could add one to test for nulls

Comment (iuhiah) [reply src/test/java/seedu/innsync/logic/commands/UntagCommandTest.java:27]: Parser passes empty strings instead of null types, command constructor will reject nulls as well

File: src/test/java/seedu/innsync/logic/parser/UntagCommandParserTest.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import org.junit.jupiter.api.Test;
+
+import seedu.innsync.logic.Messages;
+import seedu.innsync.logic.commands.UntagCommand;
+
+public class UntagCommandParserTest {
+
+    private UntagCommandParser parser = new UntagCommandParser();
+
+    @Test
```
Comment (varsha13152) [inline src/test/java/seedu/innsync/logic/parser/UntagCommandParserTest.java:18]: I think you can also test for null values
